### PR TITLE
feat(capture-sdk): Add camera access permission screen analytics

### PR DESCRIPTION
**Pull request by ndubkov-distcotech**
Ticket: PP-447

### CODE REVIEW

File: capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentImpl.java [hunk 1]
```diff
@@ -1902,4 +1912,28 @@ private void onBackPressed() {
             mCancelListener.onCancelFlow();
         }
     }
+
+    private void trackCameraNoPermissionMessageShownEvent() {
```
Comment (abolfazlimahdi) [inline capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentImpl.java:1916]: [Suggestion] I think we don't need to add functions for one line actions. What do you think? If we need them, I would suggest better names. This is the CameraAccess permission screen, not the NoPermissionMessage ....

**Review by abolfazlimahdi — COMMENTED** [review]
I have added a comment in the PR. I would also suggest that you send the status of the permission. We can talk about it if you want. Thanks :-)

File: capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentImpl.java (hunk 1 above)
Comment (ndubkov-distcotech) [reply capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentImpl.java]: Resolved

**Review by abolfazlimahdi — APPROVED** [review]
Thank you!

**Pull request merged**